metadata-app/src/components/AdditionalMetadata/index.js

```diff
@@ -32,14 +32,11 @@ const helpTexts = {
   primaryITInvestmentUIIUSG: (
     <HelpText>
       This can often be found in Exhibit 53 documents, and takes the form ###-#########. Learn more
-      about Exhibit 53 documents at{' '}
-      <a
-        target="_blank"
-        rel="noopener noreferrer"
-        href="https://www.whitehouse.gov/wp-content/uploads/2018/06/fy-2020-it-budget-guidance.pdf"
-      >
-        this pdf <LinkIcon height="15px" />
-      </a>
+      about Exhibit 53 documents at
+      {' '}
```

Comment:
I think this is extra

Reply:
If this is removed, then there is no space between the text and the link. Personally I think this is a bug in our build configuration.
<img width="338" alt="Screen Shot 2020-10-16 at 3 42 43 PM" src="https://user-images.githubusercontent.com/509703/96320124-4e81c200-0fc6-11eb-986c-0bf53f307b51.png">

Reply:
Opened https://github.com/GSA/ckanext-dcat_usmetadata/issues/108 to discuss.